Content.Client/Lobby/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -218,8 +218,24 @@ public HumanoidProfileEditor(
                 SetSpecies(_species[args.Id].ID);
                 UpdateHairPickers();
                 OnSkinColorOnValueChanged();
+                UpdateCustomSpeciesNameEdit(); // Emberfall
             };
 
+            #region Emberfall Custom Species
+
+            CustomSpeciesNameEdit.OnTextChanged += args =>
+            {
+                var text = new string(args.Text.Take(16).Where(c => char.IsLetter(c)).ToArray());
```

Comment:
I hate this. Also I think the text box should only permit letters automatically? Slicing it like I do [here](https://github.com/cosmatic-drift-14/cosmatic-drift/blob/8bc36f840846441d57ea129832e40f082a75549b/Content.Shared/_CD/Records/PlayerProvidedCharacterRecords.cs#L184-L191) should be much faster than linq but perf really does not matter for this.

Reply:
you're just a LINQ hater smh
yes idk why i went with this method where it should've just been `var newText = string.Concat(CustomSpeciesNameEdit.Text.Where(char.IsLetter));` really